doc/install.sgm

```diff
@@ -1,123 +1,101 @@
-            </para>
-        </sect1>
+ <sect1 id="install-download">
+  <title>Download</title>
 
-        <sect1 id="install-sect2">
-            <title>
-                Installation
-            </title>
-            <para>
-                You will need <application>PostgreSQL
-                9.1</application> or above. We assume that you have
-                <application>PostgreSQL</application> already compiled and
-                installed. Please note: Depending on your system configuration mostly you have to be logged in as the system
-                superuser.
-            </para>
-            <para>
-                There are two ways to compile <application>pgSphere</application>.
-                The first is to copy the sources into the contribution directory of
-                <application>PostgreSQL</application>'s source tree
-                (<filename>POSTGRESQL_SRC/src/contrib</filename>).
-                Then, change into <filename>POSTGRESQL_SRC/src/contrib</filename>.
-                If the sources are not yet installed and the directory
-                <filename>pg_sphere</filename> does not exist, take the
-                gzipped <application>pgSphere</application> sources ( e. g.,
-                <filename>pg_sphere_xxx.tgz</filename> ) and run:
-            </para>
-            <programlisting>
-<![CDATA[shell> tar -xzf path/to/pg_sphere_xxx.tgz]]>
-            </programlisting>
-            <para>
-                Now, change into the <filename>pg_sphere</filename>
-                directory and run :
-            </para>
-            <programlisting>
+  <para>
+   &pgsphere; is not the part of the &postgresql; software. You can download
+   the latest release from the
+   <ulink url="&pgsphereurl;">&pgsphere; Releases page</ulink>.
+   The source code can also be downloaded by cloning the repository with the
+   appropriate release tag. The master branch is intended for development
+   use and may contain the code in a transitional state. It is not recommended
+   for use in production.
+  </para>
+ </sect1>
+
+ <sect1 id="install-build">
+  <title>Install</title>
+  <para>
+   It is assumed that &postgresql; is already installed. Depending on the
+   system configuration, superuser (root) access rights may be required to
+   complete the installation.
+  </para>
+
+  <para>
+   The installation script uses &pg_config; utility. Make sure that the
+   environment variable PATH includes path to &pg_config; utility. The path
+   to &pg_config; can be also specified in make command:
+    <programlisting>
+<![CDATA[shell> make PG_CONFIG=/path/to/pgconfig ...]]>
+    </programlisting>
+  </para>
+
+  <para>Unpack the downloaded archive and enter the directory:</para>
+  <programlisting>
+<![CDATA[shell> tar -xzf path/to/pgsphere-X.X.X.tgz]]>
+<![CDATA[shell> cd pgsphere-X.X.X]]>
+  </programlisting>
+
+  <para>
+   Compile the code. By default, &pgsphere; is compiled with the &healpix;
```

Comment:
Please remove "the" before "&healpix;" on this line. (Sorry I didn't notice this in my previous review.)

Reply:
@esabol It's not a big deal. I've removed the article and rebased the branch. Thank you!